18F/identity-idp
# LG-13033: Reuse registration page for resending email confirmation

## 🎫 Ticket

[LG-13033](https://cm-jira.usa.gov/browse/LG-13033)

## 🛠 Summary of changes

Removes the dedicated "Resend email confirmation" screen for new user email registration in favor of redirecting the user to the "Create an account" page. This simplifies the codebase, and resolves some inconsistencies in how we validate accepted rules of use.

## 📜 Testing Plan

1. Go to http://localhost:3000/rails/mailers/user_mailer/email_confirmation_instructions
2. Click "Confirm email address"
3. Observe you are sent to the "Create an account" screen with an error message about invalid confirmation link
4. Continue account creation
5. Observe that after entering email and agreeing to Rules of Use, page submits successfully

## 👀 Screenshots

Before|After
---|---
![image](https://github.com/18F/identity-idp/assets/1779930/4e31d76c-ebb4-4be4-b519-9c436fdc236e)|![image](https://github.com/18F/identity-idp/assets/1779930/f50dc2d0-a874-481f-915d-bc7a4e21c0ef)
